service/src/test/java/bio/terra/cbas/runsets/monitoring/TestSmartRunsPoller.java

```diff
@@ -36,11 +46,29 @@ public class TestSmartRunsPoller {
   private static final String completedRunEntityId = UUID.randomUUID().toString();
   private static final OffsetDateTime completedRunSubmittedtime = OffsetDateTime.now();
 
+  public ObjectMapper objectMapper =
+      new ObjectMapper()
+          .registerModule(new ParameterNamesModule())
+          .registerModule(new Jdk8Module())
+          .registerModule(new JavaTimeModule())
+          .setDateFormat(new StdDateFormat())
+          .setDefaultPropertyInclusion(JsonInclude.Include.NON_ABSENT);
```

Comment:
Because you ended up using Gson, I suspect we no longer need this `objectMapper`?

Reply:
Got rid of it here, but had to pass in `null` as an arg for the two tests, as `SmartRunsPoller` has `objectMapper` as part of its constructor.

Reply:
So, I forgot that the test is also directly testing `SmartRunsPoller` which does in fact use `objectMapper`, so I had to put it back in an pass it as in arg in the test to get it to pass.